tyr/policies/ec2.py

```diff
@@ -18,3 +18,39 @@
         }
      ]
 }"""
+
+allow_mongo_backup_snapshot = """{
```

Comment:
Currently Mongo is the only thing that we backup using snapshots.  I could see if we do switch to doing other backups using snapshots that they would maybe require more or less privileges, such as not requiring tags.  I kind of like having it be specific to what Mongo backups need to run